doc/ch-image_desc.rst

```diff
@@ -534,6 +555,50 @@ Same, except place the image in :code:`/tmp/buster`::
    bin   dev  home  lib64  mnt  proc  run   srv  tmp  var
    boot  etc  lib   media  opt  root  sbin  sys  usr
 
+Print list of targetable architecturess for the Debian Buster image.::
+
+   $ ch-image pull --list-arch debian:buster
+   fat manifest: using existing
+   available platforms:
+   amd64
+   arm/v5
+   arm/v7
+   arm64/v8
+   386
+   mips64le
+   ppc64le
+   s390x
+
+Specify the aarch64 version of Debian Stretch using :code:`--arch`.::
+
+   $ ch-grow pull --arch=arm64/v8 debian:buster
+   pulling image:   debian:buster
+   fat manifest: using existing
+   architecture: arm64v8 (aarch64)
```

Comment:
What does it mean to have two architectures listed?

Reply:
Ah, good question. This shows us both what Docker calls the architecture and what something like `uname -m` reports. Since they can differ, e.g., above and `amd64 == x86_64`, I thought it may be helpful if both were included in the output.

Reply:
Aha. I think usability wise, we ought to stay in Docker’s vocabulary for these. Later when we’re guessing architectures we can show the translation, but here I think showing two different archs is confusing, it means we have to make the map available more broadly in the code, and going both directions will add complexity if it turns out it’s not 1-to-1.